ethereum/contracts/bridge/Bridge.sol

```diff
@@ -79,6 +79,9 @@ contract Bridge is BridgeGovernance, ReentrancyGuard {
         uint256 arbiterFee,
         uint32 nonce
     ) public payable returns (uint64 sequence) {
+        require(recipient != bytes32(0), "invalid recipient");
+        require(recipientChain != 0, "invalid chain");
```

Comment:
Similarly, there may be some reason to allow for a chain id 0 transfer and it is not prohibited in the spec